src/components/Viewport.ts

```diff
@@ -727,7 +727,7 @@ export default class Viewport {
   }
 
   public setCurrentPanel(panel: Panel): void {
-    this.currentPanel = panel.getOriginalPanel();
+    this.currentPanel = panel;
```

Comment:
is #247 no more needed?

Reply:
#247 still can be useful while in animating, but it's not that important at the moment.